net/cryfs/files/gitversion_python3_fix.diff

```diff
@@ -0,0 +1,11 @@
+--- src/gitversion/gitversion.cmake.orig	2019-07-14 22:28:20.000000000 +0200
++++ src/gitversion/gitversion.cmake	2022-04-17 15:12:06.000000000 +0200
+@@ -1,7 +1,7 @@
+ set(DIR_OF_GITVERSION_TOOL "${CMAKE_CURRENT_LIST_DIR}" CACHE INTERNAL "DIR_OF_GITVERSION_TOOL")
+ 
+ function (get_git_version OUTPUT_VARIABLE)
+-    EXECUTE_PROCESS(COMMAND python ${DIR_OF_GITVERSION_TOOL}/getversion.py
++    EXECUTE_PROCESS(COMMAND python3 ${DIR_OF_GITVERSION_TOOL}/getversion.py
```

Comment:
the `python310` port does not install (or symlink) anything to `python3`, so you cannot rely on that. Instead please change `python` to, for example, `@@PYTHONBIN@@` and then do a reinplace in the `post-patch` phase to change that to `${prefix}bin/python3.10`